spec/system/task_spec.rb

```diff
@@ -1,24 +1,22 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task, project: project) }
```

Comment:
Factoryにassociationをつけるとこの記載は省略できますね！
```suggestion
  let(:task) { create(:task) }
```